### PR TITLE
add bvt

### PR DESCRIPTION
**Pull request by doggystylez**
### Adding Assets

If adding a new asset, please ensure the following:
- [x] Asset is registered to the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry). (pending: https://github.com/cosmos/chain-registry/pull/5280)
- [x] Add asset to bottom of `zone_assets.json`.
   - [x] `chain_name` and `base_denom` are provided and use values exactly as defined at the Chain Registry.
   - [x] `path` is provided, and the IBC channel referenced is registered at the Chain Registry (unless native to Osmosis).
   - [x] `osmosis_verified` is set to `false`
   - [x] `listing_date_time_utc` is specified and accurate
   - [ ] Optional: `transfer_methods`, `peg_mechanism`, `override_properties`, `canonical`, `categories`, where necessary (see [README](https://github.com/osmosis-labs/assetlists/tree/main?tab=readme-ov-file#how-to-add-assets) for details).
- [x] I am aware that upgrading an asset to 'verified' status requires an additional PR to this repo (checklist below).

### CODE REVIEW

**Review by Cordtus — APPROVED** [review]
LGTM

File: osmosis-1/osmosis.zone_assets.json [hunk 1]
```diff
@@ -5082,6 +5082,28 @@
...
       }
+    },
+    {
+      "chain_name": "osmosis",
+      "base_denom": "factory/osmo1xu0gk9aggv79597xwazyfzaggv2pze9z7cq3p9p72tkkux9a7xaqufa792/BVT",
+      "osmosis_verified": false,
+      "listing_date_time_utc": "2024-09-07T00:00:00Z",
+      "_comment": "Banana Vault Token - Peelworks Factory II",
+      "categories": [
+        "defi",
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:5093]: how are these defi?

Comment (Cordtus) [reply osmosis-1/osmosis.zone_assets.json:5093]: It's a tokenized vault protocol.

Comment (doggystylez) [reply osmosis-1/osmosis.zone_assets.json:5093]: For context: https://forum.osmosis.zone/t/upload-banana-vaults-contracts/2484

**Pull request merged**